docs/how-to-articles/how-to-simulate-networks-with-netem.md

```diff
@@ -0,0 +1,127 @@
+Error: Specified qdisc not found.
+```
+
+In such a case, install the modules `kernel-modules-extra` and `kernel-debug-modules-extra` with following command:
+
+```
+sudo yum install kernel-debug-modules-extra kernel-modules-extra
+```
+
+After installation a reboot is needed to activate the kernel modules.
+
+## Getting Started with TC & NetEm
+
+The following examples use the interface `enp7s0`. Modify to match your network adapter (e.g. `eth0`).
+
+To be able to change settings, root privileges are required.
+
+### Add Delay
+
+**NOTE**: netem only adds delay to packets leaving the interface. If you want to simulate bi-directional delay, two instances of tc netem - one on each end - are required. 
+
+The following command adds 250 ms of delay to packets leaving interface `enp7s0`:
+
+```
+sudo tc qdisc add dev enp7s0 root netem delay 250ms
+```
+
+### Add Delay with Volatility
+
+The following command sets the transmission of enp7s0 network interface to be delayed by 100ms + 10ms (any value between 90 and 110 ms):
+
+```
+sudp tc qdisc add dev enp7s0 root netem delay 100ms 10ms
+```
+
+The volatility (randomness) of such fluctuations can be further  specified. The following command sets network interface enp7s0  transmission to 100 ms, while 30% of the packets selected at random are delayed by +10 ms:
+
+```
+sudo tc qdisc add dev enp7s0 root netem delay 100ms 10ms 30%
+```
+
+### Adding Packet Loss
+
+The following command sets the enp7s0 network interface transmission to randomly drop 1% of the packets:
+
+```
+sudo tc qdisc add dev enp7s0 root netem loss 1%
+```
+
+### Adding Packet Loss with Success Rate
+
+```
+sudo tc qdisc add dev enp7s0 root netem loss 1% 30%
+```
+
+### Duplicating Packets
+
+```
+sudo tc qdisc add dev enp7s0 root netem duplicate 1%
+```
+
+### Corrupted Packets
+
+For Linux kernel versions above 2.6.16, netem can also simulate damage to packets. The following command randomly damages 0.2% of the outgoing packets:
+
+```
+sudo tc qdisc add dev enp7s0 root netem corrupt 0.2%
+```
+
+### Packet Reordering
+
+The following command sets network interface enp7s0 to send out 25% of data packets (with 50% relevance) immediately and delay all other packets by 10 ms.
+
+```
+sudo tc qdisc add dev enp7s0 root netem delay 10ms reorder 25% 50%
+```
+
+### View the Configured Network Conditions
+
+    sudo tc qdisc show dev enp7s0
+
+### Stopping Network Impairments
+
+**IMPORTANT**: Please remember to disable the network impairments after tests are done by using the following command:
+
+```
+sudo tc qdisc del dev enp7s0 root netem
+```
+
+The above command will delete the `root` configuration that was used in the examples above.
+
+## Further Reading
+
+These were just some basic examples to get started. TC & NetEm allow complex simulations and more information can be found at following links:
+
+http://ce.sc.edu/cyberinfra/workshops/Material/NTP/Lab%204.pdf
```

Comment:
I would suggest adding the names of the documents so that there are clickable + make a list of two references.